modules/enet/networked_multiplayer_enet.cpp

```diff
@@ -496,7 +510,7 @@ uint32_t NetworkedMultiplayerENet::_gen_unique_id() const {
 					(uint32_t)OS::get_singleton()->get_unique_ID().hash64(), hash );
 		*/
 		hash = hash_djb2_one_32(
-				(uint32_t)((uint64_t)this), hash); //rely on aslr heap
+				(uint32_t)((uint64_t) this), hash); //rely on aslr heap
```

Comment:
I'm using `3.8` which is shipped with Ubuntu 16.04 LTS so maybe that's the case.
It's really strange, it's not respecting the directive `SpaceAfterCStyleCast: false` (even when forcing it).
Looks like a bug in my version.
Well, no worries I guess, I'll just be more careful until I update my system :)